Sources/TokamakCore/App/App.swift

```diff
@@ -28,22 +28,47 @@ public protocol App: _TitledApp {
   var body: Body { get }
 
   /// Implemented by the renderer to mount the `App`
-  static func _launch(_ app: Self, _ rootEnvironment: EnvironmentValues)
+  static func _launch(
+    _ app: Self,
+    with configuration: _AppConfiguration
+  )
 
   /// Implemented by the renderer to update the `App` on `ScenePhase` changes
   var _phasePublisher: AnyPublisher<ScenePhase, Never> { get }
 
   /// Implemented by the renderer to update the `App` on `ColorScheme` changes
   var _colorSchemePublisher: AnyPublisher<ColorScheme, Never> { get }
 
+  static var _configuration: _AppConfiguration { get }
+
   static func main()
 
   init()
 }
 
+public struct _AppConfiguration {
+  public let reconciler: Reconciler
+  public let rootEnvironment: EnvironmentValues
+
+  public init(
+    reconciler: Reconciler = .stack,
+    rootEnvironment: EnvironmentValues = .init()
+  ) {
+    self.reconciler = reconciler
+    self.rootEnvironment = rootEnvironment
+  }
+
+  public enum Reconciler {
+    case stack
+    case fiber(shouldLayout: Bool = false)
```

Comment:
WDYT about a slightly different naming here?

```suggestion
    case fiber(usesDynamicLayout: Bool = false)
```

Reply:
`useDynamicLayout` sounds good to me, I'll also rename `shouldLayout` in the `FiberRenderer` protocol.